### PR TITLE
Add 'webapp2' alias for 'google.appengine.ext.webapp'.

### PR DESCRIPTION
**Pull request by tseaver**
Works around: https://github.com/google/oauth2client/issues/302

### CODE REVIEW

File: gcloud/test_credentials.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def _setup_appengine_import(test_case, app_identity):
     sys.modules['google.appengine.ext'] = GAE_EXT
     sys.modules['google.appengine.ext.webapp'] = GAE_EXT_WEBAPP
     sys.modules['google.appengine.ext.webapp.util'] = GAE_EXT_WEBAPP_UTIL
+    sys.modules['webapp2'] = GAE_EXT_WEBAPP
```
Comment (dhermes) [inline gcloud/test_credentials.py]: Can we also ditch the `webapp` imports or are they required elsewhere?

**Comment by tseaver** [comment]
@dhermes PTAL:  this one blocks getting Travis going for #1128.

File: gcloud/test_credentials.py (hunk 1 above)
Comment (tseaver) [inline gcloud/test_credentials.py]: More encapsulation leakage:  auditing more deeply exactly how `oauth2client` is doing its imports, it looks like it still expects `google.appengine.ext.webapp.util` to be importable (which begs the question, "Why can we not rely then on `google.appengine.ext.webapp`?).

Comment (dhermes) [inline gcloud/test_credentials.py]: You mean why was `google.appengine.ext.webapp` deprecated in favor of `webapp2`?

If that's what you meant, the support of that module as a web framework ceased in 2011 or 2010 and `webapp2` is the replacement.

If not, I'll try to address what you did mean.

**Pull request merged**